tests/coverage/unreachable/turned-off/test.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
```

Comment:
This test is meant to check when assertion reachability checks are turned off (as indicated by its name). What is it testing for code coverage?

Reply:
Removed.